index.html

```diff
@@ -0,0 +1,300 @@
+      </div>
+      <nav class="fa fa-bars bars"></nav>
+    </header>
+    <main>
+      <section class="titleText">
+        <div class="main-container">
+          <div class="title-lp">
+            <h1>I’m Mariane Glad to see you!</h1>
+          </div>
+          <div class="titletext">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="connect-about">
+            <h2>LET'S CONNECT</h2>
+          </div>
+          <div id="footer-icon" class="footer-icons-about">
+            <ul>
+              <li>
+                <a href="#footer-icon">
+                  <img src="./images/twitter (2).svg" alt="twitter logo" />
+                </a>
+              </li>
+              <li>
+                <a href="#footer-icon">
+                  <img src="./images/Linkedin icon.svg" alt="linkedin" />
+                </a>
+              </li>
+              <li>
+                <a href="#footer-icon"
+                  ><img src="./images/m-icon.svg" alt="m logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#footer-icon"
+                  ><img src="./images/github-icon.svg" alt="github logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#footer-icon"
+                  ><img src="./images/hand-peace-icon.svg" alt="peace logo"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+    <main class="container">
+      <!-- =====TONIC-PAGE-==== -->
+      <section class="tonic1">
+        <div class="tonic-container">
+          <div class="image-tonic">
+            <img class="img-tanne" src="./images/tanne.svg" alt="" />
+          </div>
+          <div>
+            <h1 class="tonic-title">Tonic</hi>
+          </div>
+          <div class="counter">
+            <span>CANOPY</span>
+            <ul>
+              <li>
+                <img class="img-circle" src="./images/Counter.svg" alt="" /> Back
+                End Dev
+              </li>
+              <li>
+                <img class="img-circle" src="./images/Counter.svg" alt="" />2015
+              </li>
+            </ul>
+          </div>
+          <div class="tonic-para">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="list-skills">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="button" class="see-pro">See Project</button>
+        </div>
+      </section>
```

Comment:
- [ ] You correctly added the skills section, but the skills section should have a `section` tag instead of the `main` tag, with each project wrapped with a `div` tag instead of the `section` tag. For better semantics, please change that.

Reply:
Thank you and implemented